compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -13035,6 +13180,10 @@ TR::Node *ificmpgeSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
       unsignedIntCompareNarrower(node, s, TR::ifsucmpge, TR::ifbucmpge);
       }
 
+   IfcmpgeReducer ifcmpReducer(node, s);
+   if (ifcmpReducer.reducible())
+      return ifcmpReducer.reduce();
+
```

Comment:
is there no possible reason to call partialRedundantCompareElimination on the node even if we reduce it? it seems like there are other places where we just reassign node and continue along ...

Reply:
Fixed.